### PR TITLE
fix(manager/sbt): allow addCompilerPlugin

### PR DESCRIPTION
**Pull request by cjtim**
## Changes
* allow addCompilerPlugin 
* https://www.scala-sbt.org/1.x/docs/Compiler-Plugins.html

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — COMMENTED** [review]
i think this should have a test

File: lib/modules/manager/sbt/extract.spec.ts [hunk 1]
```diff
@@ -258,5 +258,21 @@ describe('modules/manager/sbt/extract', () => {
         packageFileVersion: undefined,
       });
     });
+
+    it('extract addCompilerPlugin', () => {
+      expect(
+        extractPackageFile(`
+        addCompilerPlugin("org.scala-tools.sxr" %% "sxr" % "0.3.0")
+        `)
+      ).toMatchSnapshot({
```
Comment (viceice) [inline lib/modules/manager/sbt/extract.spec.ts:267]:
```suggestion
      ).toMatchObject({
```

Comment (cjtim) [reply lib/modules/manager/sbt/extract.spec.ts]: fixed

**Review by zharinov — DISMISSED** [review]

**Comment by cjtim** [comment]
@zharinov I fix the test snapshot. please approve again thank you.

**Review by zharinov — DISMISSED** [review]

**Review by viceice — DISMISSED** [review]

**Comment by cjtim** [comment]
rebase and resolved conflict

**Review by viceice — APPROVED** [review]

**Pull request merged**